config/default/jenkinsio.yaml

```diff
@@ -12,7 +12,7 @@ ingress:
       if ( $scheme = "https" ) {
         more_clear_headers "Strict-Transport-Security";
```

Comment:
don't you need to remove this? i.e. revert https://github.com/jenkins-infra/charts/pull/98

as we now access mirrors over https

Reply:
I'll move the HSTS stuff on another PR